src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -23,6 +23,8 @@
 import net.java.sip.communicator.service.protocol.event.*;
 import net.java.sip.communicator.util.Logger;
 
+import org.jitsi.impl.protocol.xmpp.extensions.*;
+import org.jitsi.jicofo.*;
```

Comment:
I thought that we were intentionally avoiding depending on jicofo classes in impl.protocol.xmpp. Is there an important reason to introduce it?
cc @paweldomas

Reply:
I guess it would be better if JitsiMeetConference would subscribe to some start muted thing instead of ChatRoomImpl setting it here.

Reply:
@bgrozev Well, I couldn't know about that.
@paweldomas Not sure about the best way to do that. How about letting `JitsiMeetConferenceImpl` subscribe to presence events of `ChatRoomImpl`? Or something in `ChatRoomRoleAndPresence`? I think I was considering the latter, but didn't go for it since the necessary information (i.e. the actual presence packet to access the extension) isn't present. So a proper presence listener in `ChatRoomImpl` would still be necessary.

Reply:
So the problem is that there's no longer way to add PacketListener to the connection directly ? I need to take a look.

Reply:
You can still add StanzaListeners, but not for get/set IQs (well, you can in Smack 4.2, but no longer in 4.3).